janusgraph-test/src/test/java/org/janusgraph/diskstorage/util/StaticArrayEntryTest.java

```diff
@@ -39,14 +39,22 @@ public void testToString() {
     }
 
     @Test
-    public void testToStringWithNoValue() {
-        long l = random.nextLong();
+    public void testRandomLongToStringWithNoValue() {
+        testToStringWithNoValue(random.nextLong());
+    }
+
+    @Test
+    public void testHexLongWithLeadingZeroToStringWithNoValue() {
+        testToStringWithNoValue(242385170064350798L);
```

Comment:
Shall we use a less "magical" number here, e.g. `Long.MAX_VALUE >> 3` (which has 15 "F"s in hex format)?

Reply:
I got this value from one of the failing tests but you are right, `Long.MAX_VALUE >> 3` is less confusing. Changed that to `Long.MAX_VALUE >> 3`